scripts/setup-ubuntu.sh

```diff
@@ -83,9 +83,16 @@ function install_folly {
   cmake_install -DBUILD_TESTS=OFF
 }
 
+function install_conda {
```

Comment:
This change seems not relevant. Can it be extracted into a separate PR?

Reply:
Thanks Mahsa, this is from another PR, rebasing with main will remove this.